README.md

```diff
@@ -25,7 +25,7 @@ The contents of this repository are protected by Crown Copyright (C).
 ### Requirements
 * make
 * nodejs + npm/yarn
-* [poetry](https://github.com/python-poetry/poetry)
+* Python 3.8 + [poetry](https://github.com/python-poetry/poetry)
```

Comment:
Out of curiosity - do we use anything introduced in v3.9 that would cause us to need to bump this?

Reply:
I would like the python version to be 3.12 but I couldn't do it easily due to the common pipelines provided by APIM using Python 3.8 -> https://github.com/NHSDigital/api-management-utils/tree/master

Reply:
I don't think we need to bump the version yet due to Python 3.8 being currently supported and everything works with Python 3.8